README.md

```diff
@@ -10,5 +10,66 @@ make install
 make start
 ```
 
+## Requirements
+
+Application requires few other services and configuration in environment
+variables. Docker-compose in this repository is configured to read variables
+from `.env` file that is created by `make init` command.
+
+### Keycloak
+
+In the Keycloak admin console create a new client for the API with following
+configuration
```

Comment:
You may wish to start by creating a separate 'realm' in keycloak for the Resource Hub.  If so, also create an admin account which can be used by the API to make authenticated requests to keycloak.  Then within the realm create a new client for the API with the following configuration: